.github/workflows/php.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         run: ./vendor/bin/phpunit --coverage-clover build/clover.xml
 
       - name: PHP CS Fixer
-        run: ./vendor/bin/php-cs-fixer fix --dry-run
+        run: PHP_CS_FIXER_IGNORE_ENV=1 ./vendor/bin/php-cs-fixer fix --dry-run
```

Comment:
Just curious: what is the goal of this `PHP_CS_FIXER_IGNORE_ENV` ?

Reply:
In fact, PHP CS Fixer isn't really ready for 8.2 yet and a PHP version check is done before scan. With `PHP_CS_FIXER_IGNORE_ENV` we disable this check and we force CS fixer to run!

Reply:
👍